de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewRepresentation.java

```diff
@@ -61,6 +63,22 @@ public void setTable(final JSONDataTable table) {
     m_table = table;
   }
 
+  public String getRemoteRepositoryURL() {
+    return m_remoteRepositoryURL;
+  }
+
+  public void setRemoteRepositoryURL(String m_remoteRepositoryURL) {
```

Comment:
Regarding the naming of the parameters of setRemoteRepositoryURL I think we should rename them to simply remoteRepositoryURL and maxSelectionNumber. The point of the m_ prefix is to indicate class variables and this can be counterintuitive as the parameters are not class variables.